src/System.Management.Automation/FormatAndOutput/common/TableWriter.cs

```diff
@@ -596,5 +584,7 @@ private static string GenerateRowField(string val, int width, int alignment, Dis
         private bool _hideHeader = false;
 
         private int _startColumn = 0;
+
+        const int columnsThresHold = OutCommandInner.columnsThresHold;
```

Comment:
Same here. No need to declare another const int variable.

Reply:
Fixed.